docs/index.md

```diff
@@ -12,6 +12,8 @@ It also supports a rich set of higher-level tools including [Spark SQL](sql-prog
 
 # Downloading
 
+{% include_example scala/org/apache/spark/examples/SparkALS.scala SparkALS %}
```

Comment:
Could turn it into a real example including Scala/Java/Python? We can use the pipeline example http://spark.apache.org/docs/latest/ml-guide.html#example-pipeline.